docker-compose.yml

```diff
@@ -8,6 +8,34 @@ services:
     hostname: coreapi-postgres
     environment:
       POSTGRESQL_USER: coreapi
-      POSTGRESQL_PASSWORD: coreapipostgres
+      POSTGRESQL_PASSWORD: coreapi
       POSTGRESQL_DATABASE: coreapi
-    container_name: coreapi-postgres
\ No newline at end of file
+    container_name: coreapi-postgres
+
+  alembic_migrations:
+    build:
+      context: .
+      dockerfile: Dockerfile.migrations
+    network_mode: bridge
```

Comment:
I believe you also want to set `restart: on-failure` as there is no guarantee that postgres container will be up and running before migration script gets executed.